addon/components/frost-popover.js

```diff
@@ -12,13 +13,16 @@ export default Component.extend(PropTypeMixin, {
   layout,
   classNameBindings: ['visible:visible:invisible', 'autoPosition'],
   classNames: ['tooltip-frost-popover'],
+
   propTypes: {
     closest: PropTypes.bool,
     delay: PropTypes.number,
+    hideDelay: PropTypes.number, // This currently doesn't work properly with 'click'
```

Comment:
@helrac can you track this with an issue, once we merge this?

Reply:
Yeah, i can do that. I believe we may want to expand this component as well, allow events for the originating target and the content target and make those all customizable. Like, maybe you want mouse leave to have a delay, but clicking away is instant, or hitting escape or something also instant.